src/wp-admin/includes/class-wp-upgrader.php

```diff
@@ -937,6 +943,25 @@ public static function release_lock( $lock_name ) {
 		return delete_option( $lock_name . '.lock' );
 	}
 
+	/**
+	 * Gets the hook extra action.
+	 *
+	 * @since 5.6.0
+	 *
+	 * @param string $package_type The package type (plugin or theme).
+	 * @return string hook extra action type. Default is "install".
+	 */
+	protected function get_hook_extra_action( $package_type ) {
+		if (
+			! empty( $this->skin->overwrite )
+			&&
+			in_array( $this->skin->overwrite, array( "update-{$package_type}", "downgrade-{$package_type}" ), true )
```

Comment:
Can we change this to just `update` and `downgrade`? The `$hook_extra` parameter should also contain a `$type`, which will be `plugin`, `theme`, or `core`.